biggraphite/drivers/cassandra.py

```diff
@@ -2042,9 +2061,17 @@ def run(rows):
 
         token = start_token
         while token < stop_token:
-            rows = self._execute_metadata(select, (token,), DEFAULT_TIMEOUT_QUERY_UTIL)
-            if len(rows.current_rows) == 0:
-                break
+            try:
+                rows = self._execute_metadata(select, (token,), DEFAULT_TIMEOUT_QUERY_UTIL)
+
+                # Empty results means that we've reached the end.
+                if len(rows.current_rows) == 0:
```

Comment:
if I am not wrong you can simply write
if not rows.current_rows:

Reply:
I'd like to not change this code (I just moved it). I think I remember there was a reason for the len() but I'm not sure which one.

Reply:
ok